easybuild/easyblocks/s/star_ccm.py

```diff
@@ -69,23 +106,54 @@ def install_step(self):
 
         env.setvar('IATEMPDIR', tempfile.mkdtemp())
 
-        cmd = ' '.join([
-            self.cfg['preinstallopts'],
-            install_script,
-            "-i silent",
-            "-DINSTALLDIR=%s" % self.installdir,
-            "-DINSTALLFLEX=false",
-            "-DADDSYSTEMPATH=false",
-            self.cfg['installopts'],
-        ])
+        # argument -DINSTALLFLEX is -DINSTALL_LICENSING for the .aol installer
+        if self.aol_install:
+            cmd = ' '.join([
+                self.cfg['preinstallopts'],
+                # The install_script installs also the Siemens Installer Program (SIP)
+                # under $HOME, this is not need to run STAR-CCM+
+                'HOME=%s' % self.builddir,
+                install_script,
+                "-i silent",
+                # for some reason the installation directory's name cannot be the version
+                # So using builddir and then moving to installdir
+                "-DINSTALLDIR=%s" % self.builddir,
+                "-DINSTALL_LICENSING=false",
+                "-DADDSYSTEMPATH=false",
+                self.cfg['installopts'],
+                "&& mv %s/%s* %s" % (self.builddir, self.version, self.installdir),
```

Comment:
the easyblock already does the move in line 148, isn't it?
```
run_shell_cmd("mv " + os.path.join(self.builddir, entry) + ' ' + self.installdir)
```

I like that`run_shell_cmd` more than appending a `mv` to this installation command, so if this one is indeed redundant please remove.